### ISSUE TITLE
The Error "Mod 'Ambiance' (ambiance) 1.1.0 requires any version of configuration, which is missing!" is confusing

### ISSUE THREAD

**Issue by Vuyisce** [ChorusTeam/Ambiance#6]
Im not sure if this is an major issue but everytime I try to start up minecraft after putting Ambiance into my mods folder. It pops up with an error that says "Mod 'Ambiance' (ambiance) 1.1.0 requires any version of configuration, which is missing!". What configuration is it talking about? To I have to install another mod for it to work? Any tips will be helpful

**Comment by ghost** [comment]
yeah configuration is a mod https://modrinth.com/mod/configuration
